src/components/Node/Node.js

```diff
@@ -87,19 +87,20 @@ export default function Node({
         )}
         {thumbnails[0] && (
           <span>
-          {showFace && faceImage ? (
-
-          <img
-            alt={faceImage.alt }
-            src={faceImage.url+ (imageType === 'head' ? '?factor=1.5' : '')}
-            title={faceImage.alt}
-          />
-          ) : (
-            <img
-              alt={thumbnails[0].alt}
-              src={thumbnails[0].url}
-              title={thumbnails[0].alt}
-            />
+            {showFace && faceImage ? (
+              <img
+                alt={faceImage.alt}
+                src={
+                  faceImage.url + (imageType === "head" ? "?factor=1.5" : "")
+                }
+                title={faceImage.alt}
+              />
+            ) : (
+              <img
+                alt={thumbnails[0].alt}
+                src={thumbnails[0].url}
+                title={thumbnails[0].alt}
+              />
```

Comment:
you've still not enabled prettifier on your IDE

Reply:
I don't know how 😭